v2/clients/http/deviceprofile_test.go

```diff
@@ -68,3 +70,59 @@ func TestPutDeviceProfiles(t *testing.T) {
 	require.NotNil(t, res)
 	require.Equal(t, requestId, res[0].RequestId)
 }
+
```

Comment:
Add negative error tests when file is not found.

Reply:
Added negative test to check file not found error.